content/components/relative-time.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Relative time
+status: Experimental
+---
+
+import {Box, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A relative time displays the relative time difference between the source date-time and the provided date-time.
+</Box>
+
+## Content
+
+### Past
+
+For past dates, the relative time outputs different formats depending on the time difference.
+
+Within the current month, the output shows a relative date in the format of `X seconds/minutes/hours/days ago`. If the time difference is less than 1 minute, it shows `just now`.
+
+After a month has passed, the output shows a relative date in the format `on D MMM` or `on D MMM, YYYY` depending on the current year.
```

Comment:
```suggestion
After a month has passed, the output shows a relative date in the format `on D MMM` if the date is within the current year or `on D MMM, YYYY` if the date is within the previous year or before.
```

I hope this makes it more clear.

Reply:
What about this? 

```suggestion
After a month has passed, the output shows a relative date in the format `on D MMM` if the date is within the current year or `on D MMM, YYYY` if the date is past the current year.
```

Reply:
I am not 100% sure "past the current year" is correct (language wise). 🤔

Reply:
🤔  what about using the structure we have in `micro` or `future` descriptions? 

Besides this, I'd follow @maximedegreve comment and add a table that simplifies and helps with these definitions.

```suggestion
If the date is in the following months, the format `on D MMM` is used. If the date is in the following years, the format `on D MMM, YYYY` is used.
```